app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt

```diff
@@ -24,6 +24,7 @@ import timber.log.Timber
  * Created by collinx on 22-10-2017.
  */
 
+@Suppress("DEPRECATION")
```

Comment:
Why suppress?

Reply:
`androidx.test.runner.AndroidJUnit4` was showing deprecation there and without this the circle ci build was also failing.

Reply:
There should be another way

Reply:
okay @liveHarshit I will try this with some different approach

Reply:
@liveHarshit made the changes as asked by you